modules/seqexec/server/src/main/scala/seqexec/server/ghost/GhostController.scala

```diff
@@ -81,6 +103,80 @@ sealed trait GhostConfig {
   def configuration: Configuration = ifu1Config |+| ifu2Config
 }
 
+// These are the parameters passed to GHOST from the WDBA.
+// We use them to determine the type of configuration being used by GHOST, and instantiate it.
+object GhostConfig {
+  private[ghost] def ifuConfig(ifuNum: IFUNum,
+                        ifuTargetType: IFUTargetType,
+                        coordinates: Coordinates,
+                        bundleConfig: BundleConfig): Configuration = {
+    def cfg[P: GiapiConfig](paramName: String, paramVal: P) =
+      Configuration.single(s"${ifuNum.configValue}.$paramName", paramVal)
+      cfg("target", ifuTargetType.configValue) |+|
+      cfg("type", DemandType.DemandRADec.demandType) |+|
+      cfg("ra", coordinates.ra.toAngle.toDoubleDegrees) |+|
+      cfg("dec", coordinates.dec.toAngle.toDoubleDegrees) |+|
+      cfg("bundle", bundleConfig.configValue)
+  }
+
+  private[ghost] def ifuPark(ifuNum: IFUNum): Configuration = {
+    def cfg[P: GiapiConfig](paramName: String, paramVal: P) =
+      Configuration.single(s"${ifuNum.ifuStr}.$paramName", paramVal)
+      cfg("target", IFUTargetType.NoTarget.targetType) |+|
+      cfg("type", DemandType.DemandPark.demandType)
+  }
+
+  def apply(baseCoords: Option[Coordinates],
+            expTime: Duration,
```

Comment:
This is the part I'm questioning: we no longer have a single `expTime`, but two `expTimes`: one for the red camera and one for the blue camera, along with exposure counts. We also have spectral / spatial binning for each of the two cameras, and read / gain.

Reply:
Do we need to pass these through the seqexec?

Reply:
(Definitely the `expTime` seems like it should be removed, I would think.)

Reply:
I expect that we send on the sequence the blue and red times and a total exp time (or maybe have a way to calculate it in the seqexec). We need a total exp time to set the timeouts